woocommerce/sensei-course-progress
# Add Hungarian translation

Dependent on #82.

This adds Hungarian translation files.

## Testing
1. In _Settings_ > _General_, change the _Site Language_ to Hungarian (i.e. Magyar).
2. Go to _Appearance_ > _Widgets_.
3. Look for the _Sensei LMS - Course Progress_ widget.
4. Ensure the description is translated.
5. Add the widget to the sidebar.
6. Ensure the widget checkbox setting is translated.
